src/theory/quantifiers/sygus/cegis_unif.cpp

```diff
@@ -36,43 +38,54 @@ bool CegisUnif::initialize(Node n,
                            std::vector<Node>& lemmas)
 {
   Trace("cegis-unif") << "Initialize CegisUnif : " << n << std::endl;
-  Assert(candidates.size() > 0);
-  if (candidates.size() > 1)
+  /* For regular CEGIS */
```

Comment:
This is also copy/pasted code right? This should be encapsulated somewhere.

Reply:
actually, you can just remove this code, this is for cegis-sample, which is likely incompatible with cegisUnif.